repo/packages/P/prometheus/0/config.json

```diff
@@ -0,0 +1,22 @@
+{
+  "type": "object",
+  "properties": {
+    "prometheus": {
+      "description": "Prometheus specific configuration properties",
+      "type": "object",
+      "properties": {
+        "master": {
```

Comment:
a required property is framework-name nested under the service name.   something like `prometheus.framework-name = "prometheus"`  then use that for the default of the marathon app id.